bin/wsdump.py

```diff
@@ -116,6 +116,8 @@ def recv_ws():
         while True:
             opcode, data = recv()
             msg = None
+            if six.PY3 and type(data) is bytes:
```

Comment:
Oh! Yes, I'm sorry I was wrong using ENCODING. About the TEXT type, you are right but if it is not a TEXT type is converted to string too. Anyway, I amended it with your suggestion.